src/controllers/execute.ts

```diff
@@ -46,31 +47,32 @@ export const execute = async (ctx: Context): Promise<void | Context> => {
     desc: '',
   };
 
-  const _ = request.networkMap.messages.map((messages) => {
-    return messages.channels.map((channels) => {
-      return channels.typologies.map((typologies) => {
-        /* eslint-disable array-callback-return */
-        return typologies.rules.map((rule) => {
-          if (rule.id === ruleRes.id) {
-            ruleRes.cfg = rule.cfg;
-            return ruleRes;
+  ruleRes.cfg = (() => {
+    for (const messages of request.networkMap.messages) {
+      for (const channels of messages.channels) {
+        for (const typologies of channels.typologies) {
+          for (const rule of typologies.rules) {
+            if (rule.id === ruleRes.id) {
+              return rule.cfg;
+            }
           }
-        });
-      });
-    });
-  });
+        }
+      }
+    }
+    return '';
```

Comment:
instead of return '' here, we could throw an error. If this rule is not in the networkmap, might as well error out. 
```suggestion
    throw new Error(`Rule ${ruleRes.id} not found in network map`);
```